components/TokenManagePage/Media.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import { useTokenProvider } from "@/providers/TokenProvider";
+import { useState, useEffect } from "react";
+import { Input } from "@/components/ui/input";
+import { Textarea } from "@/components/ui/textarea";
+
+const Media = () => {
+  const { metadata } = useTokenProvider();
+  const { data: meta, isLoading } = metadata;
+
+  const [formData, setFormData] = useState({
+    title: "",
+    description: "",
+    image: "",
+    animationUrl: "",
+  });
```

Comment:
keep state simple
- actual: state defined as `object`
- required: 2 distinct state. one for `title` and other for `description`